angrmanagement/ui/main_window.py

```diff
@@ -122,6 +124,11 @@ def open_newstate_dialog(self):
         new_state_dialog = NewState(self.workspace.instance, parent=self)
         new_state_dialog.exec_()
 
+    def open_doc_link(self):
+        QDesktopServices.openUrl(QUrl("https://docs.angr.io/", QUrl.TolerantMode))
+
+    def open_about_dialog(self):
+        QMessageBox.about(self, "About angr", "version 8.19.2.4")
```

Comment:
I would remove the version number because we don't want to hard code `8.19.2.4` here.